src/pages/docs/infrastructure/deployment-targets/index.mdx

```diff
@@ -30,7 +30,7 @@ A deployment target is a location that will host your software. We've used the t
 - Serverless functions
 - SSH connections
 
-Deployment targets represent any intended destination for the contents in your package. You can also run day-2 operations tasks against the deployment targets you set up in Octopus.
+Deployment targets therefore define where your software is deployed, and they also serve as the basis for Day-2 tasks and additional operational tasks that can be managed by Octopus.
```

Comment:
I think we can remove the word "therefore" to simplify it a little, but otherwise a nice improvement.

```suggestion
Deployment targets define where your software is deployed, and they also serve as the basis for Day-2 tasks and additional operational tasks that can be managed by Octopus.
```